goldmane/proto/methods.go

```diff
@@ -21,6 +21,8 @@ import (
 
 	"github.com/sirupsen/logrus"
 	"k8s.io/kubernetes/pkg/apis/core/validation"
+
+	"github.com/projectcalico/calico/libcalico-go/lib/names"
```

Comment:
Did we intentionally hard coded prefixes in this implementation instead of using the const defined in the `names` package? Using it add dependency between goldmane and libcalico-go. Wondering if this was by purpose.

Reply:
I think it was unintentional, but also largely because this code was / is considered a bit throwaway. Fine to introduce the libcalico import.